projects/js-packages/charts/src/components/line-chart/line-chart.tsx

```diff
@@ -488,9 +490,13 @@ const LineChartInternal = forwardRef< SingleChartRef, LineChartProps >(
 							onPointerOut={ onPointerOut }
 							pointerEventsDataKey="nearest"
 						>
-							<Grid columns={ false } numTicks={ 4 } />
-							<Axis { ...chartOptions.axis.x } />
-							<Axis { ...chartOptions.axis.y } />
+							{ showGrid && <Grid columns={ false } numTicks={ 4 } /> }
+							{ showAxes && (
+								<>
+									<Axis { ...chartOptions.axis.x } />
+									<Axis { ...chartOptions.axis.y } />
+								</>
+							) }
```

Comment:
How about adding a `display` property to the existing chart options `AxisOptions`
```suggestion
							{ chartOptions.axis.x.display && <Axis { ...chartOptions.axis.x } /> }
							{ chartOptions.axis.y.display && <Axis { ...chartOptions.axis.y } /> }
```

Reply:
Obviously needs to default to true

Reply:
Oh yup, that's a good idea. Thanks :)